docs/user-event/api-utility.mdx

```diff
@@ -0,0 +1,179 @@
+---
+id: utility
+title: Utility APIs
+---
+
+The following APIs don't have one-to-one equivalents in a real user
+interaction.  
+Their behavior is therefore an interpretation how the "perceived" user
+interaction might be translated to actual events on the DOM.
```

Comment:
I'm not sure, but maybe this is better?

```suggestion
This means that their behavior is our own interpretation of how the browser behaves and the eventing mechanism should not be taken as a source of truth.
```

Reply:
I think that would be misleading. The problem is not the browser behavior, it's the user interaction.
There is not *one* specific user interaction that is described by those APIs. So we can not reproduce the *one* correct behavior of the API. So we roll with what *some* (or *most*?) users think that user interaction would be and then implement the correct behavior for this.
But in reality there are either different input methods to reach the described interaction or there simply is no actual interaction like described by the API and perceived by our users.

Reply:
Yeah I understand what you're saying here. :)